src/poliastro/core/maneuver.py

```diff
@@ -0,0 +1,85 @@
+""" Low level manuever implementations """
+
+import numpy as np
+from numba import njit as jit
+from numpy import cross
+from numpy.linalg import norm
+
+from poliastro.core.elements import coe_rotation_matrix, rv_pqw
+
+
+@jit
+def hohmann(k, r_i, r_f, v_i, inc, raan, argp, nu, ecc):
```

Comment:
I think we could just pass the following parameters:

`hohmann(k, rv, r_f)`

This simplifies the hihg-level API even more and ensures all computations are carried out inside de jitted functions.